draft-secure-credential-transfer.md

```diff
@@ -652,6 +652,15 @@ the Sender MUST append the Secret as URI fragment {{!RFC3986}}, so that the resu
 ~~~
 {: #link-with-fragment title="Example of URL with Secret as URI Fragment"}
 
+## Second factor authentication for Receiver credential provisioning
+
+- Provisioning Partner shall require an additional security confirmation (PIN code) from Receiver Device at the time of credential provisioning.
+- PIN code shall be generated by the Sender Device at the time when it creates a new Mailbox with Provisioning Information on a Relay Server.
+- PIN code shall be sent from Sender device to Receiver device in a secure way (preferrably over encrypted channel) out-of-band with the Mailbox URL and Secret
+- If Sender device can not send the PIN code over secure channel, it may include it into encrypted Payload stored on the relay server so that Receiver device can decrypt it and use to get Provisioning Information from Provisioning Partner.
```

Comment:
I think this is more about if the sender is ok with decreased security they can store the PIN code in the encrypted payload. Not about the available sending channels. Even when only one channel is available the sender could still delay sending the PIN code until the receiver tries to accept.

Reply:
The point here is that PIN, being a 2-factor, needs to be sent over a different channel (what you have - phone number in addition to 1st factor secret - what you know). If out of band PIN delivery is not available, PIN transforms to the 1st factor - secret (and we just allow to encrypt it along with the mailbox payload).